src/Aws/General.hs

```diff
@@ -204,12 +207,26 @@ parseRegion =
     <|> UsEast1 <$ P.text "us-east-1"
     <|> UsWest1 <$ P.text "us-west-1"
     <|> UsWest2 <$ P.text "us-west-2"
+    <|> CustomEndpoint . T.pack <$> (P.text "custom:" *> many P.anyChar)
```

Comment:
:wink: Well, I'm fine with taking responsibility for it, since I wrote the code... I have to work on some other things for the next bit of time, so if you get impatient, feel free to add whatever you need to this PR.

Otherwise, when I have finished my other tasks, I will come back and add them.